package.json

```diff
@@ -18,6 +18,7 @@
   "dependencies": {
     "axios": "^1.6.7",
     "pinia": "^2.1.7",
+    "universal-cookie": "^7.1.0",
```

Comment:
is er een specifieke reden voor deze package? :)

Reply:
Had ik toegevoed om een probleem proberen op te lossen, maar ben blijkbaar vergeten te verwijderen.